.github/workflows/e2e-main.yaml

```diff
@@ -120,7 +120,6 @@ jobs:
         working-directory: ./podman-desktop-extension-ai-lab
         env:
           PODMAN_DESKTOP_ARGS: ${{ github.workspace }}/podman-desktop
-          AI_LAB_CI_RUN: true
```

Comment:
What was the reason of this env ?

Reply:
it was removed, because it's referencing another draft PR. my bad, I shouldn't have merged it in the first place (prior PR)